aimmo-game/simulation/avatar/avatar_manager.py

```diff
@@ -28,7 +28,6 @@ def get_avatar(self, user_id):
 
     def remove_avatar(self, user_id):
         if user_id in self.avatars_by_id:
-            self.avatars_to_delete_by_id[user_id] = copy.deepcopy(self.avatars_by_id[user_id])
```

Comment:
is this self.avatars_to_delete_by_id still used/useful in the avatar manager?

Reply:
This is also done in #223 I will remove it here.

Reply:
It should have been called, actually. I got confused as we don't do it anymore in #223